### DISCUSSION TITLE
How do I change the color of a notification in React-Native?

### DISCUSSION THREAD

**Discussion by nurassyl** [invertase/react-native-firebase#5559] [Q&A] [answered]
**How do I change the color of a notification in React-Native?**

When I receive a notification, I want to change the background color, which is blue and inside my logo.
[Screenshot](https://i.stack.imgur.com/Xt3qH.jpg)

**Comment by mikehardy** [comment]
I believe the information you want, at least for Android, is here? https://firebase.google.com/docs/reference/fcm/rest/v1/projects.messages#androidnotification

**Comment by nurassyl** [comment, marked as answer]
I solved this problem by changing my app icon background color, it will get dominant color of app icon.
